api/can_api_v2_definition.py

```diff
@@ -121,6 +133,45 @@ class ActualsTimeseriesRow(Actuals):
     date: datetime.date = pydantic.Field(..., description="Date of timeseries data point")
 
 
+class MetricAnomaly(base_model.APIBaseModel):
+    date: datetime.date = pydantic.Field(..., description="Date of anomaly")
+    description: str
+
+
+class MetricAnnotations(base_model.APIBaseModel):
+    """Annotations associated with one field."""
+
+    sources: List[MetricSources]
+    anomalies: List[MetricAnomaly]
```

Comment:
My initial thought was that would all be optional fields - as we add more, I think some metrics definitely won't have annotations? Was this a specific decision?